### PR TITLE
Adding Obsolete Attributes to missing type/members

### LINKED ISSUE #13741: Add missing [Obsolete] attributes on NS2.0 ports
We should have any [Obsolete] attributes that are present in Standard. Some are missing. Copy them from the Standard repo. 

These are missing:

* various on AppDomain
* various on Process
* various on Thread
* these two
```
    public class HttpWebResponse : WebResponse, ISerializable {
        [ObsoleteAttribute("Serialization is obsoleted for this type.  http://go.microsoft.com/fwlink/?linkid=14202")]
        protected HttpWebResponse(SerializationInfo serializationInfo, StreamingContext streamingContext);
    }
    public class IPAddress {
        [ObsoleteAttribute("This property has been deprecated. It is address family dependent. Please use IPAddress.Equals method to perform comparisons. http://go.microsoft.com/fwlink/?linkid=14202""This property has been deprecated. It is address family dependent. Please use IPAddress.Equals method to perform comparisons.http://go.microsoft.com/fwlink/?linkid=14202")]
        public long Address { get; set; }
    }
```

### PR DESCRIPTION
**Pull request by joperezr**
Fixes #13741 

cc: @weshaggard @danmosemsft 

Adding obsolete members in order to match netstandard2.0

### CODE REVIEW

**Comment by danmosemsft** [comment]
Per https://github.com/dotnet/corefx/pull/13745#discussion_r88574116 these can break serialization. However none of these seem like types that get serialized.
@stephentoub

File: src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs [hunk 1]
```diff
@@ -310,6 +310,7 @@ public long NonpagedSystemMemorySize64
             }
         }
```
Comment (joperezr) [inline src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs]: you are right, I did miss a few. I was doing my comparison against bin\layout\ref\netstandard1.7 which for some reason doesn't have System.Diagnostics.Process which is why I missed it in the first pass. This time I used bin\layout\ref\netcoreapp1.1 and got all of those missing ones.

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop CentOS7.1 Release Build and Test  (log gone)

**Comment by joperezr** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please (WinHttpHandler tests failed)

**Comment by danmosemsft** [comment]
@mmitche 
```
javax.servlet.ServletException: org.apache.commons.jelly.JellyTagException: jar:file:/var/cache/jenkins/war/WEB-INF/lib/jenkins-core-1.642.1.jar!/hudson/model/AbstractBuild/index.jelly:110:84: <st:include> java.lang.NullPointerException
	at org.kohsuke.stapler.jelly.JellyClassTearOff.serveIndexJelly(JellyClassTearOff.java:117)
```

@dotnet-bot test Innerloop Windows_NT Debug Build and Test

**Comment by mmitche** [comment]
@danmosemsft Yeah, saw that.  My fix was a bit bad, but I figured out what it was and worked around it.  Shouldn't hit this time.

**Pull request merged**